src/map/utils/trustutils.cpp

```diff
@@ -654,6 +700,61 @@ namespace trustutils
             sCHR = 0;
             sVIT = 0;
         }
+        
+        if (mLvl >= 99 && mLvl < 105)
+        {
+            mSTR = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 2), mLvl) * 1.2;
+            mDEX = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 3), mLvl) * 1.2;
+            mVIT = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 4), mLvl) * 1.2;
+            mAGI = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 5), mLvl) * 1.2;
+            mINT = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 6), mLvl) * 1.2;
+            mMND = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 7), mLvl) * 1.2;
+            mCHR = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 8), mLvl) * 1.2;
+        }        
+        
+        if (mLvl >= 105 && mLvl < 110)
+        {
+            mSTR = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 2), mLvl) * 1.5;
+            mDEX = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 3), mLvl) * 1.5;
+            mVIT = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 4), mLvl) * 1.5;
+            mAGI = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 5), mLvl) * 1.5;
+            mINT = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 6), mLvl) * 1.5;
+            mMND = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 7), mLvl) * 1.5;
+            mCHR = mobutils::GetBaseToRank(grade::GetJobGrade(PTrust->GetMJob(), 8), mLvl) * 1.5;
+        }        
+        
+        if (mLvl >= 110 && mLvl < 115)
```

Comment:
This condition duplicates the above, should be combined

Reply:
adjusted the upper range to not be duplicitous.